_includes/manuals/1.23/1.2_release_notes.md

```diff
@@ -24,6 +24,48 @@ The UI team has been hard at work behind the scenes at making our new React-base
 #### Experimental Redis cache support
 Redis has been added as an experimental caching backend for Foreman instead of the default file-based store, or the memcache-based store which requires an additional plugin. It is still missing installer support and documentation - contributions are most welcome! ([#26918](https://projects.theforeman.org/issues/26918))
 
+### Release notes for 1.23.2
+#### Authentication
+* Wrong Error Message/Return code getting after exceeding the 'failed_login_attempts_limit' ([#28860](https://projects.theforeman.org/issues/28860))
+
+#### Compute resources
+* Custom field on host view ([#29059](https://projects.theforeman.org/issues/29059))
+
+#### Compute resources - oVirt
+* NoMethodError: undefined method \`storage_domain' for nil:NilClass ([#26919](https://projects.theforeman.org/issues/26919))
+* oVirt API v4 incorrectly enabled VM options ([#24924](https://projects.theforeman.org/issues/24924))
+
+#### Database
+* Deprecate SQLite production support ([#28248](https://projects.theforeman.org/issues/28248))
+
+#### Debian/Ubuntu
```

Comment:
While reading this, I wonder if we should rename this in Redmine to `Debian/Ubuntu packaging`